Applied modified patch for extra output options

akydd opened the pull request:
Added new optional parameters for the `diatheke` command:
* `--no-refs` suppresses reference printing in output
* `--no-mod-name` suppresses the module name in output
* `--no-newlines` replaces new lines with a space in output

I am not the author of these changes; the original diff is here:
http://www.crosswire.org/pipermail/sword-devel/attachments/20130326/5c7fd8bd/attachment.obj

The code base had changed since that diff was created. This PR Implemented identical functional changes, but to the code base in GitHub.

vonpupp commented:
Greetings,

In first place I would like to thanks for the great idea diatheke+sword is, nevertheless for a console user I think that the current diatheke output is not very pipe-friendly as it could be, which is such a pity since with all the CLI tools available, new tools and processes could be conceived using pipes.

I would suggest to accept/merge the proposed new options by @raphink so diatheke can be more powerful and flexible on CLI usage.

Thank you very much for the excelent work. God bless you!

Commit:
Applied modified patch for extra output options

Original diff:
http://www.crosswire.org/pipermail/sword-devel/attachments/20130326/5c7fd8bd/attachment.obj

Code had changes since that diff was created. Implemented identical
functional changes to the updated code.

greg-hellings commented:
This repository is only a mirror of https://crosswire.org/svn/sword

Please submit all patches to the sword-devel mailing list or the CrossWire JIRA tracker.
